src/install/config/ibm-mq.yaml

```diff
@@ -0,0 +1,61 @@
+agentName: ibm-mq
+agentType: integration
+title: 'IBM MQ monitoring integration'
+metaDescription: 'Install the IBM MQ integration'
+introFilePath: 'src/install/ibm-mq/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'linux'
+        displayName: 'Linux'
+        logo: 'linux'
+      - value: 'windows'
+        displayName: 'Windows'
+        logo: 'windows'
+      - value: 'tarball'
+        displayName: "Install from tarball (Other Linux distros)"        
+        logo: 'linux'  
+
+steps:
+  - filePath: 'src/install/ibm-mq/appInfo.mdx'
+  - filePath: 'src/install/ibm-mq/default-requirements.mdx'
+    overrides: 
+        - filePath: 'src/install/ibm-mq/linux/requirements-linux.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'linux'
+        - filePath: 'src/install/ibm-mq/windows/requirements-windows.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'windows'
+        - filePath: 'src/install/ibm-mq/linux/requirements-tarball.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'tarball'                
+  - filePath: 'src/install/ibm-mq/default-install.mdx'
```

Comment:
This doc is missing the step where the users pull down/download the actual integration. More info in comments below